src/sqs/github-app-config-payload.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Types for both Cloud and Enterprise server product
+ */
+export type GitHubAppConfigPayload = {
+	/*
+	 * undefined for cloud, real value for GHE
+	 * Otherwise, lost of duplicates values in msg body for cloud (majority traffic)
+	 */
+	gitHubAppConfig?: {
+		appId: number,
+		clientId: string,
+		gitHubBaseUrl: string,
+		uuid: string,
+		gitHubAppId: string,
+		gitHubClientSecret: string,
+		webhookSecret: string,
+		privateKey: string
```

Comment:
Does this mean we would add the `gitHubClientSecret`, `webhookSecret`, and `privateKey` to every backfill and push SQS message? Why do we need those? Can't we just add the subscription ID to the message and then load the secrets when (and if) we need them in the message workers?

Reply:
I don't think we need the webhook secret for this one specifically as that's specific to webhooks only.

The idea behind having everything in a config is that _everything_ in the code uses the github app config (API, webhooks and sync), so we reduce complexity by removing as much logic as possible from the processors.  This is standard practice for queues message as you want all information to process the job at hand with the least amount of logic.  By adding logic, you're creating another layer of complexity that also makes it much more difficult to update in the future.  

If we are to use this methodology, we can simply pass the config to any client or handlers that use it.